DesignSafe-CI/portal
# Task/WG-316 Data Depot disable menu hazmapper

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [WG-316](https://tacc-main.atlassian.net/browse/WG-316)

## Summary of Changes: ##

## Testing Steps: ##
1. Select a .hazmapper file and make sure the copy, move, etc file options are disabled
2. Open the preview modal of a hazmapper file and ensure it opens in a new tab and closes the modal


TODO:

- [ ] Factor out hazmapper extension type from text common settings

## UI Photos:
![Screenshot 2024-07-12 at 3 31 23 PM](https://github.com/user-attachments/assets/11fdc68c-6d46-46c1-bc0c-5e4f802f0a3a)




## Notes: ##
Not sure how to share a hazmapper map with you locally. I am not able to add you to projects I have locally. Let me know if you have any issues finding one.
